main.go

```diff
@@ -46,6 +48,13 @@ type options struct {
 	WorkflowConfigPath              string
 }
 
+func (o *options) Validate() error {
+	if err := o.GitHubOptions.Validate(false); err != nil {
+		return err
+	}
+	return nil
+}
+
```

Comment:
Technically this is `return o.GitHubOptions.Validate(false)` ;)

Reply:
Done!